src/broker/broker.c

```diff
@@ -813,6 +813,11 @@ receiver_thr_f (void *arg)
 #endif
 
 #if defined(LINUX)
+  if (br_shard_flag == ON)
```

Comment:
How about sleeping in broker.c:575 in main(),  rather than sleeping in the receiver_thr_f() function ?
And I would like to know why it sleeps 700 ms only in Linux environment.

Reply:
my intention is:
start receiver_thr_f, but it will sleep few msecs and shard_dispatch_thr will start immediately. If we put sleep at line 575, this will postpone both threads for few msecs and start shard_dispatch_thr_f as soon as receiver_thr_f starts

For the 700ms, it has no special meaning. It can be predicted that it will take some time as it has to run db login. It was desired that the value be less than 1 second. The actual application results were also positive, so it was used.

This process is necessary for very few special applications, and sleep itself may not be necessary.
.

Reply:
I changed it as a macro, and changed value to 900. Because it only wait once at the shard broker startup,
it could not be burden. If you have opinion, comments, let me know it.

#define SHARD_CAS_WARMUP_MSEC 900